api/src/main/java/uk/gov/dhsc/htbhf/claimant/reporting/payload/ReportPropertiesFactory.java

```diff
@@ -101,16 +138,29 @@ public ReportPropertiesFactory(@Value("${google-analytics.tracking-id}") String
         customMetrics.put(CHILDREN_UNDER_ONE.getFieldName(), childrenUnder1);
         customMetrics.put(CHILDREN_BETWEEN_ONE_AND_FOUR.getFieldName(), childrenUnder4 - childrenUnder1);
         LocalDate expectedDeliveryDate = claimant.getExpectedDeliveryDate();
-        if (expectedDeliveryDate == null || expectedDeliveryDate.isBefore(atDate)) {
-            customMetrics.put(PREGNANCIES.getFieldName(), 0);
-        } else {
+        if (isClaimantPregnant(expectedDeliveryDate, atDate)) {
             customMetrics.put(PREGNANCIES.getFieldName(), 1);
-            LocalDate conception = expectedDeliveryDate.minusWeeks(PREGNANCY_DURATION_IN_WEEKS);
-            customMetrics.put(WEEKS_PREGNANT.getFieldName(), ChronoUnit.WEEKS.between(conception, atDate));
+        } else {
+            customMetrics.put(PREGNANCIES.getFieldName(), 0);
         }
         return customMetrics;
     }
 
+    private boolean isClaimantPregnant(LocalDate expectedDeliveryDate, LocalDate atDate) {
```

Comment:
Do we not need to use the pregnancy calculator for this?

Reply:
No, Karen confirmed that for reporting purposes we want to be using this simple approach.